configure.ac

```diff
@@ -146,9 +146,9 @@ AC_CACHE_SAVE
 # during 'make distcheck'.
```

Comment:
Good point.  I'm pretty new to automake so I feel pretty lobotomized when working on it.